test/extended/pods/priorityclasses.go

```diff
@@ -0,0 +1,52 @@
+package pods
+
+import (
+	"context"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = Describe("[sig-arch] Managed cluster should", func() {
+	oc := exutil.NewCLIWithoutNamespace("pod")
+
+	It("ensure platform components have system-* priority class associated", func() {
+		// iterate over the references to find valid images
+		pods, err := oc.KubeFramework().ClientSet.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{})
+		if err != nil {
+			e2e.Failf("unable to list pods: %v", err)
+		}
+
+		// list of pods that use images not in the release payload
+		invalidPodPriority := sets.NewString()
+		// a pod in a namespace that begins with kube-* or openshift-* must come from our release payload
```

Comment:
This needn't be true for `openshift-*` namespaces as some OLM managed operators user this also.

Reply:
If you're just referring to the comment, I don't mind removing it. However please note that this test suite when run in origin wouldn't include those OLM managed components unless explicitly deployed.

Reply:
Yes I am just referring to the comment. You can reword it to clarify given you are suggesting in your email that OLM managed operators can also pull in this test.